ballerina/task_errors.bal

```diff
@@ -17,3 +17,9 @@
 # Represents the error type of the `ballerina/task` module. This error type represents any error that can occur during
 # the execution of the task APIs.
 public type Error distinct error;
+
+# Represents an error that occurs when a task scheduling operation fails.
+public type SchedulingError distinct Error;
+
+# Represents an error that occurs when a task execution operation fails.
+public type ExecutionError distinct Error;
```

Comment:
Could you please explain the usage of these two error types ? What does a SchedulingError has which can not be captured in `Error` or `ExecutionError`. If we don't have any specific use-case its better to use the Error type.

Reply:
Fixed in [ebb6ca0](https://github.com/ballerina-platform/module-ballerina-task/pull/534/commits/ebb6ca028751707bacdada8cffc50e8b7edb6c70)